src/api/folder.js

```diff
@@ -0,0 +1,16 @@
+const userPath = "https://bootcamp-api.codeit.kr";
+
+export const folderInfo = async () => {
+  try {
+    const response = await fetch(`${userPath}/api/sample/folder`);
+
```

Comment:
```suggestion
export const folderInfo = async (api) => {
  try {
    const response = await fetch(`${userPath}/api/${api}`);

```

fetch와 관련된 로직을 분리해서 사용할 때는, 특정 api만 사용되게 분리하는 것이 아닌
파라미터로 api주소와 option 값을 받을 수 있도록 구성해야 합니다.